dashboard/modules/metrics/metrics_head.py

```diff
@@ -318,10 +322,35 @@ async def record_dashboard_metrics(self):
             Component=self._component,
             SessionName=self._session_name,
         ).set(float(dashboard_proc.memory_full_info().uss) / 1.0e6)
+        self._dashboard_head.metrics.metrics_event_loop_lag.labels(
+            ip=self._ip,
+            pid=self._pid,
+            Component=self._component,
+            SessionName=self._session_name,
+        ).set(float(self.event_loop_lag_s))
 
     async def run(self, server):
         self._create_default_grafana_configs()
         self._create_default_prometheus_configs()
+        head = self
+
+        def on_new_lag(lag_s):
+            # Keep the lag. It's exported in `record_dashboard_metrics`
+            head.event_loop_lag_s = lag_s
+
+        enable_monitor_loop_lag(on_new_lag)
+
+        # Logs warnings if a task ran for more than 100ms. Limitation: it only logs
+        # the task name and duration, e.g. if the blocker is in a aiohttp handler it
+        # only logs `RequestHandler._handle_request()` instead of the actual handler.
+        enable_log_slow_callbacks(
+            slow_duration=0.1,
```

Comment:
Should we make it configurable and have a way to disable it?

Reply:
added flag, default to disabled